src/table/__tests__/selections-util.spec.js

```diff
@@ -181,13 +183,30 @@ describe('selections-utils', () => {
       const params = ['/qHyperCubeDef', [2, 1], [cell.colIdx]];
       const payload = {
         colIdx: cell.colIdx,
-        newRows: [
+        selectedRows: [
           { qElemNumber: 2, rowIdx: 2 },
           { qElemNumber: 1, rowIdx: 1 },
         ],
       };
+      const event = { ctrlKey: false, metaKey: false };
 
-      selectCell(cell, selState, selDispatch);
+      selectCell(cell, selState, selDispatch, event);
+      expect(selState.api.begin).to.not.have.been.called;
+      expect(selState.api.select).to.have.been.calledWith({ method: 'selectHyperCubeCells', params });
+      expect(selDispatch).to.have.been.calledWith({ type: 'select', payload });
+    });
+    it('should not call begin, ctrl/command keyboard to add to selected and call selectHyperCubeCells when selecting new qElemNumber', () => {
```

Comment:
Since we refactored, I think we should test `getSelectedRows` separately. Then make some simpler tests for `selectCell`, which basically tests the different cases, not that that `selectedRows` is containing the right stuff. We can merge this as is, and I can fix it later, just me being picky about unit test structure